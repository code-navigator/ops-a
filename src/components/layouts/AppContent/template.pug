v-content#nav

  //- Inject pages here
  router-view