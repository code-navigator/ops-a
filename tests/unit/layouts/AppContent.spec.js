import AppContent from '@/components/layouts/AppContent/'
import Vue from 'vue'
import VueRouter from 'vue-router'
import Vuetify from 'vuetify'
import { mount, createLocalVue } from '@vue/test-utils'
import { TestHelpers } from '../TestHelpers'

Vue.use(Vuetify)

describe('layout: AppContent.vue', () => {
  const localVue = createLocalVue()
  const name = 'AppContent'
  const router = new VueRouter()
  let h, wrapper

  localVue.use(VueRouter)

  // Run these statements before each test
  beforeEach(() => {
    wrapper = mount(AppContent, {
      router,
      localVue
    })

    h = new TestHelpers(wrapper, expect)
  })

  // Is component mounted?
  test('should mount without errors', () => {
    h.isInstance()
  })

  // Does component have correct name?
  test('has name', () => {
    h.hasName(name)
  })

  // Does component render correct template
  test('renders correctly', () => {
    h.renders()
  })
})
