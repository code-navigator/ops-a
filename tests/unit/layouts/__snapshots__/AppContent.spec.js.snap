// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout: AppContent.vue renders correctly 1`] = `
<main class="v-content" style="padding: 0px 0px 0px 0px;" id="nav">
  <div class="v-content__wrap"></div>
</main>
`;
